### ISSUE TITLE
Map disappears after hot reload

### ISSUE THREAD

**Issue by UsamaKarim** [liodali/osm_flutter#77]
Hi, I am using example code, When I hot reload the app, the map disappears

![image](https://user-images.githubusercontent.com/64292174/118482233-e1860480-b72d-11eb-8e4f-27632757b5c9.png)

**Comment by liodali** [comment]
yeah the osm map in android for now doesn't support hot relaod but i will try to fix those problem as soon as possible

**Comment by UsamaKarim** [comment]
What about rotation of the map?

**Comment by liodali** [comment]
the same also,their is problem between lifecycle of osm view and flutter lifecycle

**Comment by liodali** [comment]
this bug will be fixed in `0.20.0`
